app/presenters/user_presenter.rb

```diff
@@ -0,0 +1,31 @@
+class UserPresenter < SimpleDelegator
+  include ActionView::Helpers::FormOptionsHelper
+
+  delegate :global, to: :user_roles, prefix: true
+
+  def self.wrap(users)
+    users.map { |user| new(user) }
```

Comment:
I wondered about this too. I don't see it being a problem in how it's being used so far (in `GlobalUserRolesController` and `UserFinder`), but I'm sure it's something we'll run into as we do more decorating/presenting in this project. Do you know a good pattern for this? I'd consider putting it in a base presenter class.